feat: Enhance E2E tests with inventory update scenarios and storeAvailability feature (#227)
algolia/algoliasearch-sfcc-b2c: CONTRIBUTING.md

# Contributing to Algolia Salesforce B2C Commerce

This document explains the steps required to get the project running, run the automated test-suite, and submit changes in a way that is easy for the maintainers to review.

## 1. Prerequisites

Before you start, make sure you:

- Access to a **Salesforce B2C Commerce sandbox** that contains the *RefArch* storefront
- Have an **Algolia application** with an Admin API key.

## 2. Project layout (quick tour)

- `cartridges/` – server-side cartridge sources.
- `cypress/` – end-to-end test suite.
- `scripts/` – helper scripts run by npm tasks.
- `test/` - includes test files

## 3. Environment variables

The test-suite and some scripts rely on a few secrets that **must not be committed to the repository**.
Create a `.env` file at the project root (it is already in `.gitignore`) by using `.env.example` file.

### 3.1 Creating the test shopper (one-time)

Storefront User credentials
```
"test@algolia.com",

> "password":"Algolia2025!"

Author: Removed them

# Contributing to Algolia Salesforce B2C Commerce

This document explains the steps required to get the project running, run the automated test-suite, and submit changes in a way that is easy for the maintainers to review.

## 1. Prerequisites

Before you start, make sure you:

- Access to a **Salesforce B2C Commerce sandbox** that contains the *RefArch* storefront
- Have an **Algolia application** with an Admin API key.

## 2. Project layout (quick tour)

- `cartridges/` – server-side cartridge sources.
- `cypress/` – end-to-end test suite.
- `scripts/` – helper scripts run by npm tasks.
- `test/` - includes test files

## 3. Environment variables

The test-suite and some scripts rely on a few secrets that **must not be committed to the repository**.
Create a `.env` file at the project root (it is already in `.gitignore`) by using `.env.example` file.

### 3.1 Creating the test shopper (one-time)

> Storefront User credentials
> ```
> "test@algolia.com",
> "password":"Algolia2025!"
> ```
> 
> Cypress logs in with the email/password above.
> If that account doesn't exist on your sandbox yet, you can create it:

Reviewer: You also need to update the guide to reflect that:
```suggestion
Cypress will log in to the Storefront with the email/password defined in `TEST_SHOPPER_EMAIL` and `TEST_SHOPPER_PASSWORD`.
If no test account exist on your sandbox yet, you can create one:
```
